src/main/java/nl/novi/spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,42 @@
+package nl.novi.spring_boot_tech_it_easy_controller.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class TelevisionsController {
+
+
+    @GetMapping("/televisions")
+    public ResponseEntity<String> getAllTelevisions() {
+//        response return 200 status with string
+        return ResponseEntity.ok("televisions");
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<String> getTelevisionbyId(@PathVariable int id) {
+//        response return 200 status with specific string id
+        return ResponseEntity.ok("television with id: " + id);
+    }
+
+    @PostMapping("/televisions")
+    public ResponseEntity<String> createTelevision(@RequestBody String television) {
+        // television creation
+        //return 201 response status and a string
+        return ResponseEntity.created(null).body("televisions");
+    }
+
+    @PutMapping("/televisions/{id}")
+    public ResponseEntity<String> updateTelevision(@RequestBody String television) {
```

Comment:
Jou pad is "/television/{id}" die {id}  is een PathVariable. Je moet in de parameters van deze methode vervolgens wel een `@Pathvariable int id` definieren om die waarde uit het pad te kunnen uitlezen.

Het idee van de PutMapping is dat je een specifieke tv pakt met een specifiek id en die aanpast met de waardes die je in de RequestBody hebt ontvangen.